molecule/default/molecule.yml

```diff
@@ -42,6 +42,12 @@ platforms:
     privileged: true
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
+  - name: clearlinux
+    image: paulfantom/clearlinux-molecule:latest
```

Comment:
👋 @paulfantom, I'm not sure if you'll be using the `latest` tag here. Happy to update this to whatever you see fit.

Reply:
latest is ok